demo-vue/src/validators/formValidator.js

```diff
@@ -0,0 +1,19 @@
+import { form, min, required, compare, every } from 'strickland';
+import { formDefinition } from '../../../demo/src/formValidator';
+
+const formDefinitionExtended = {
+  ...formDefinition,
```

Comment:
I think I'd be more comfortable if you just copied the content from the react demo `formValidator` instead of extending it.  That way we can keep the demos separated from one another.

Reply:
Sure thing, I assumed you might want to extend your form to be the same as the one I've created in the vue demo (to test out other field types) and we'd re-align, but will separate for now.